ppmclibs/tinycv.xs

```diff
@@ -1,227 +0,0 @@
-#ifndef _GNU_SOURCE
```

Comment:
where did this file end?

Reply:
restored and did a different approach by still calling all the commands for ppmclibs as in before, just from cmake.